docs/modules/reference/pages/glossary.adoc

```diff
@@ -128,13 +139,15 @@ Minions communicate with these remote devices while OpenNMS performs coordinatio
 Nephron:: OpenNMS component that enables horizontal scaling of flow processing.
 Not required for flow processing, but can improve performance for deployments with a very high volume of flow documents.
 
-Network Operating Center:: A Network Operating Center (NOC) is one or more locations where a network is monitored and controlled.
+Network Operating Center (NOC):: One or more locations where a network is monitored and controlled.
 (See https://en.wikipedia.org/wiki/Network_operations_center[Network Operations Center].)
 
 Newts:: Newts (New-Fangled Time Series Datastore) is an OpenNMS time series data store based on Apache Cassandra.
 Use it as an alternative persistence strategy instead of JRobin or RRDtool.
 (See xref:deployment:time-series-storage/newts/introduction.adoc#ga-opennms-operation-newts[Newts].)
 
+Northbounder:: An alarm generated by {page-component-title} that is sent to a higher-level component (service, resource, etc.).
```

Comment:
I'm not an expert on northbounding, but my perception of it was sending an object from one Horizon/Meridian instance to another, external system, whether it's another H/M instance or something else.

Reply:
I'll ask for clarification on this one. :)